metaworld_algorithms/rl/networks.py

```diff
@@ -214,7 +214,7 @@ class QValueFunction(nn.Module):
     config: QValueFunctionConfig
 
     @nn.compact
-    def __call__(self, state: jax.Array, action: jax.Array) -> jax.Array:
+    def __call__(self, state: jax.Array, action: jax.Array, mask: jax.Array | None = None, store: bool = False) -> jax.Array:
```

Comment:
The RL network interfaces should be generally agnostic to lower level implementation details. The only thing that should be necessary here is taking in a network hidden state or something to make it work with recurrent architectures.